html/ops/create_forums.php

```diff
@@ -29,7 +29,7 @@
 
 function create_category($orderID, $name, $is_helpdesk) {
     $q = "(orderID, lang, name, is_helpdesk) values ($orderID, 1, '$name', $is_helpdesk)";
-    $db = BoincDB::get();
+    $db = BoincDb::get();
```

Comment:
The case was incorrect. PHP is case-insensitive but it is better to us the correct casing.